pkg/apis/operators/v1alpha1/servicebinding_types.go

```diff
@@ -47,6 +47,14 @@ type ServiceBindingSpec struct {
 	BindAsFiles bool `json:"bindAsFiles,omitempty"`
 }
 
+// ServiceBindingMapping defines a mapping from the existing collection of Secret values to a new Secret entry.
+type ServiceBindingMapping struct {
+	// Name is the name of the mapped Secret entry
+	Name string `json:"name"`
+	// Value is the value of the new Secret entry.
```

Comment:
We should state here that the value is actually template and rendered value gets injected into application.